src/libraries/System.IO.Packaging/src/System/IO/Packaging/ContentType.cs

```diff
@@ -351,7 +351,7 @@ private void ParseParameterAndValue(ReadOnlySpan<char> parameterAndValue)
         /// <returns></returns>
         private static int GetLengthOfParameterValue(ReadOnlySpan<char> s, int startIndex)
         {
-            Debug.Assert(s != null);
+            Debug.Assert(s != default);
```

Comment:
Changing null to default here isn't meaningful.

This assert should just be deleted.